### PR TITLE
Added moderation support 

### PR DESCRIPTION
**Pull request by ishevche**
It resolves issue #1.

Every listing, before being sent to a specified channel, is sent to a moderator, defined by its CHAT_ID in the `.env` file. 

For each listing, a moderator has two options: approve and decline. If the moderator approves the listing, it is posted to the channel, and the seller is notified with a message containing a button to revoke the listing. If the listing is declined, only a notification for the seller is sent.

Six newly created text variables are used: 
 - `bot.listing_approved{listing}` - notification for seller with listing approval
 - `bot.listing_declined{listing}` - notification for seller with listing approval
 - `bot.moderator_approval` - text displayed on the approve button for the moderator
 - `bot.moderator_declination` - text displayed on the decline button for the moderator
 - `bot.approved_successfully` - text contained in approving callback_query answer
 - `bot.declined_successfully` - text contained in declining callback_query answer
 - `bot.send_to_moderation` - replacement for `thanks_sale` string, sent to a user when a newly created listing is sent to a moderator for check

The first two variables accept listing - a text of the message displayed in the channel.

### CODE REVIEW

**Review by Salz0 — CHANGES_REQUESTED** [review]

**Comment by mykolasolodukha** [comment]
@ishevche please merge the latest `master` into your feature branch.

**Review by mykolasolodukha — CHANGES_REQUESTED** [review]
@ishevche please merge the latest master into your feature branch.

**Review by Salz0 — CHANGES_REQUESTED** [review]
Hey, thanks again for this awesome PR, please review the suggestions I've put

File: keyboards.py [hunk 1]
```diff
@@ -25,3 +30,34 @@
...
+async def cancel_listing_keyboard(channel_message_id):
+    # Cancel listing markup for seller
+    reply_markup = InlineKeyboardMarkup()
+    reply_markup.add(
+        InlineKeyboardButton(
+            i18n.gettext("bot.cancel_sell", locale=BOT_LANGUAGE),
+            callback_data=f"cancel {channel_message_id}",
+        )
+    )
+    return reply_markup
```
[the remark this answers is not in the document]
Comment (ishevche) [reply keyboards.py:63]: For each user and each listing, different keyboards have to be created, with the ID of the corresponding listing bound with this keyboard. As a result, each instance has to be created dynamically during the program's lifecycle, so they can not be initialized once the program starts as keyboards above.

[the remark this answers is not in the document]
Comment (Salz0) [reply keyboards.py:63]: Agreed
One question remains: Why have it async instead of simply def?

**Comment by Salz0** [comment]
Great, thank you man for this big PR and for sticking to it util it's done. Highly appreciated 🙌

**Pull request merged**